client/me/site-blocks/main.jsx

```diff
@@ -72,9 +73,9 @@ class SiteBlockList extends Component {
 					<p>
 						{ translate(
 							'Blocked sites will not appear in your Reader and will not be recommended to you.'
-						) }
-						<a href="https://en.support.wordpress.com/reader/#blocking-sites">
-							{ translate( ' Learn more.' ) }
+						) }{ ' ' }
```

Comment:
Is that extra whitespace (`{ ' ' }`) intended?

Reply:
Looks good to me. Translatable strings should not begin or end with a whitespace and there is one needed between the sentence and the Learn more.